checkstyle/README.adoc

```diff
@@ -224,3 +224,19 @@ Documentation and further configuration for each check can be found https://chec
   </executions>
 </plugin>
 ----
+
+=== Use file filter BeforeExecutionExclusionFileFilter to exclude files from being processed by the utility
```

Comment:
please create separate heading and text for it, === is a level 3 heading.

Reply:
Changed to level 4 heading and improved text in commit d4aa53e760a47d8003f64d970162090bf7ed3239